src/ontology/mondo-edit.obo

```diff
@@ -258328,6 +258356,7 @@ name: autosomal recessive limb-girdle muscular dystrophy type 2N
 def: "Autosomal recessive limb-girdle muscular dystrophy type 2N (LGMD2N) is a form of limb-girdle muscular dystrophy characterized by proximal weakness (manifesting as slowness in running) presenting in infancy, along with calf hypertrophy, mild lordosis, scapular winging and normal intelligence or mild intellectual disability." [Orphanet:206559]
 subset: ordo_disease {source="Orphanet:206559"}
 synonym: "autosomal recessive limb-girdle muscular dystrophy caused by mutation in POMT2" EXACT [MONDO:design_pattern]
+synonym: "LGMD-POMT2 related" EXACT CLINGEN_PREFERRED []
```

Comment:
could you please add a dbxref here:
https://clinicalgenome.org/affiliation/50061/